_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,66 @@
+hidden: false
+section: software-applications
+
+---
+
+## Things that you will need
+
+- 1-2 flash drives (1GB and up)
+- Philips Screwdriver
+- Approx. 30 mins.
+
+## The error messages
+
+If you are seeing any of these messages on your Open Firmware system:
+
+```
+configuring bootloader error: command failed with exit status: exit code: 5
+```
+
+```
+errored while installing system: command failed with exit status: exit code: 5
+```
+
+```
+calling Schedule method failed: "failed to add boot entry: exit code: 5
+```
+
+```
+system76-firmware: failed to schedule: failed to add boot entry: exit code: 5
+```
+
+we may need to clear the SMMSTORE ROM by flashing new firmware using a flash drive with a custom firmware update tool. Here are the steps to perform that action. After flashing, this will remove any custom EFI variables (such as those installed by boot managers). We have information on repairing the bootloader at the end of this article. 
+
+Please download the zip file for your sytem and unzip the files within to a FAT32 formatted USB drive. Then shutdown the laptop and remove all NVMe M.2 drive(s) in the system using the step by step instructions for your system:
+
+|   System            |              Patched Firmware                            |                                        Tech Docs                                            |
+|:-------------------:|:--------------------------------------------------------:|:-------------------------------------------------------------------------------------------:|
+| Bonobo WS  (bonw14) | [bonw14-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip) | [bonw14](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd) |
+| Darter Pro (darp6)  | [darp6-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [darp6](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Darter Pro (darp7)  | [darp7-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [darp7](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Galago Pro (galp4)  | [galp4-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [galp4](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Galago Pro (galp5)  | [galp5-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [galp5](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Lemur Pro (lemp9)   | [lemp9-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [lemp9](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
+| Lemur Pro (lemp10)  | [lemp10-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip) | [lemp10](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd) |
+| Oryx Pro (oryp6)    | [oryp6-76ec.zip](/files/clear-smmstore/bonw14-76ec.zip)  | [oryp6](https://tech-docs.system76.com/models/bonw14/repairs.html#replacing-an-m2nvme-ssd)  |
```

Comment:
Am I wrong, or do all these links point to the same bonw14-76ec.zip file?

Reply:
Ah yea.